txpipe/twopoint_fourier.py

```diff
@@ -563,6 +559,14 @@ def compute_power_spectra(
                     0,
                     CEE,
                 ),
+                (
```

Comment:
After you've made all your fixes could you run `black` on this file to tidy the layout?  Actually on all the changed files.